invoicer_project/clients/views.py

```diff
@@ -53,6 +51,12 @@ def get_user_from_jwt(headers):
     return current_user['user_refr'].to_dict()
 
 
-@api_view(['GET'])
-def home_view(request):
-    return render(request, 'clients/clients_list.html', context={"first_name": "", "last_name": ""})
+class ClientRenderingViewSet(viewsets.ViewSet):
+    @action(detail=False, methods=["get"])
+    def home_view(self, request):
+        return render(request, 'clients/clients_list.html', context={"first_name": "", "last_name": ""})
```

Comment:
It looks like the context doesn't need for this template

Reply:
Done